### PR TITLE
Completed exercise

### PR DESCRIPTION
**Pull request by gabriellerramos**
Screen Shot 2022-09-27 at 9 30 14 PM

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var firstElement = 'the first elemenent of the array is html';
+console.log(firstElement);
+var thirdElement = 'the 3rd element of the array is javascript';
+console.log(thirdElement);
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js]: We can access elements of an array by their index.

```js
var firstElement = languages[0];
```

File: prep-javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+var length = 'The length of the array is 6';
+console.log(length);
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js]: We can read the length of any array from its `.length` property.

```js
var length = languages.length;
```

File: prep-javascript-arrays/main.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+var lastElement = 'the last element of the array is postgres';
+console.log(lastElement);
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js]: We can calculate the last index of an array by subtracting 1 from its length and then using the result to access the last element of the array.

```js
var lastElement = languages[languages.length - 1];
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by gabriellerramos** [comment]
Screen Shot 2022-10-03 at 2 23 35 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**